src/Simplify.cpp

```diff
@@ -525,7 +538,11 @@ class Simplify : public IRMutator {
         const Select *select_a = a.as<Select>();
         const Select *select_b = b.as<Select>();
 
-        if (const_int(a, &ia) &&
+        if (is_error_call(call_a)) {
```

Comment:
Bounds inference is probably the most likely case to be concerned about for the new code, too: a human is unlikely to deliberately write this sort of code (we only found it from a random fuzz-simplify test), but it makes sense that bounds inference could come up with this nonsense and get a wrong answer. (That said: seems like the signed integer overflow *could* happen elsewhere as well, it's just unlikely for a human to write.)